privacyidea/api/lib/prepolicy.py

```diff
@@ -74,14 +81,25 @@
                                    parse_timedelta, is_true, check_pin_policy, get_module_class)
 from privacyidea.lib.crypto import generate_password
 from privacyidea.lib.auth import ROLE
-from privacyidea.api.lib.utils import getParam
+from privacyidea.api.lib.utils import getParam, attestation_certificate_allowed
 from privacyidea.lib.clientapplication import save_clientapplication
 from privacyidea.lib.config import (get_token_class, get_from_config, SYSCONF)
 import functools
 import jwt
 import re
 import importlib
+
 # Token specific imports!
+from privacyidea.lib.tokens.webauthn import (WebAuthnRegistrationResponse, AUTHENTICATOR_ATTACHMENT_TYPES,
+                                             USER_VERIFICATION_LEVELS, ATTESTATION_LEVELS)
+from privacyidea.lib.tokens.webauthntoken import (WEBAUTHNACTION, DEFAULT_PUBLIC_KEY_CREDENTIAL_ALGORITHM_PREFERENCE,
```

Comment:
It would probably be nicer to just move all this code into the token itself (which would spare us alot of complexity), but this is an architectural change we should adopt for all token-specific policies.